build/Targets/RepoToolset/Readme.md

```diff
@@ -0,0 +1,6 @@
+Files in this directory are copied from RepoToolset. They might be slightly modified to adjust for the current Roslyn build system specifics if necessary.
+
+Limit Roslyn specific changes to minumum.
+
+If a fix not specific to Roslyn is needed create a PR against https://github.com/dotnet/roslyn-tools repo updating the toolset. Update files in this directory only if the change to RepoToolset gets approved and merged.
```

Comment:
It may be wise to put this as a comment header in the files themselves. I don't think people will notice the readme.

Reply:
OK.

---
In reply to: [147290592](https://github.com/dotnet/roslyn/pull/22850#discussion_r147290592) [](ancestors = 147290592)